### ISSUE TITLE
[v128] Junk Log Dialog Window and View Problem

### ISSUE THREAD

**Issue by kiki-ja** [Betterbird/thunderbird-patches#337]
Betterbird 128.1.0ESR 64bit
(Also Thunderbird 128.1.0ESR 64bit)
Windows 10 Pro 22H2 64bit

Does anyone else have a small window size and closed log area as shown in the screenshot below?

![Settings - Junk - Show log - Dialog](https://github.com/user-attachments/assets/51b7518f-b650-434a-bed1-ed446bdea362)

![Adaptive Junk Mail Log](https://github.com/user-attachments/assets/5ba6b75f-3323-4955-98ee-bf8325aefc17)

------------

I have fixed it in the following userChrome.css.
I hope this will be helpful.

```
@charset "utf-8";
/* Junk Mail Log Window size and Log view custom */
#viewLogWindow {
 min-width: 600px !important; 
 min-height: 375px !important; 
}
#logView {
 min-width: 570px !importnat; 
 min-height: 290px !important; 
}
```
![Adaptive Junk Mail Log -1](https://github.com/user-attachments/assets/4f73469e-75e7-40e9-adc3-6bd27daf5121)

@kiki-ja

**Comment by Betterbird** [comment]
This issue was reported as https://bugzilla.mozilla.org/show_bug.cgi?id=1691301 and we've incorporated the fix in Betterbird 128.1.0esr-bb6 "Preview" (build 8) (15 August 2024):
https://github.com/Betterbird/thunderbird-patches/blob/ce5dc7f8c490b07b1ca675af93612a7567d2c51f/128/series#L163

Are you using that version?

According to the upstream bug report, version 115 is also affected, and we've already discussed this in Issue #203. Maybe we should backport the fix also to the last version in the 115 series, 115.15.0, due in early September.
